target/linux/ramips/dts/mt7620a_dlink_dwr-961-a1.dts

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * D-Link DWR-961 A1 Board Description
+ * Copyright 2022 Pawel Dembicki <paweldembicki@gmail.com>
+ */
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+#include <dt-bindings/leds/common.h>
```

Comment:
Can we drop this now?

Reply:
Yeah, I was pretty sure that I removed it.